garage/tf/regressors/deterministic_mlp_regressor_model.py

```diff
@@ -0,0 +1,72 @@
+"""DeterministicMLPRegressorModel."""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.models import MLPModel
+
+
+class DeterministicMLPRegressorModel(MLPModel):
```

Comment:
isn't this more accurately `NormalizedInputMLPModel`? I bet you could also make an outer model `NormalizeInputs` which wraps any model and does this.

Reply:
I agree with the naming. However, the `network_input_spec` and `network_output_spec` for different model is different. We can't have a single model `NormalizeInputs` which represents all the model with normalized inputs since their inputs/outputs are different. Would it be too complicated if we create a base model `NormalizeInputs` and have each model inherit from it and add specific `network_input_spec` and `network_output_spec`?

Reply:
SGTM

Reply:
+1 on `NormalizedInputMLPModel`. I was refactoring `CategoricalMLPRegressor` and it uses the exact same subclass as this.